cli_tools_e2e_test/gce_image_import_export/test_suites/import/import_tests.go

```diff
@@ -230,55 +230,52 @@ var basicCases = []*testCase{
-			requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-linux-uefi-rhel-7-from-image",
-			// image created from projects/gce-uefi-images/global/images/rhel-7-v20200403 and removed UEFI_COMPATIBLE
-			source:                  "projects/compute-image-tools-test/global/images/linux-uefi-no-guestosfeature-rhel7",
-			os:                      "rhel-7",
-			requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-linux-nonuefi-debian-9",
-			// source created from projects/debian-cloud/global/images/debian-9-stretch-v20200714
-			source:                    "gs://compute-image-tools-test-resources/uefi/linux-nonuefi-debian-9.vmdk",
-			os:                        "debian-9",
-			notAllowedGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-linux-hybrid-ubuntu-1804",
-			// source created from projects/gce-uefi-images/global/images/ubuntu-1804-bionic-v20200317
-			source:                  "gs://compute-image-tools-test-resources/uefi/linux-hybrid-ubuntu-1804.vmdk",
-			os:                      "ubuntu-1804",
-			osConfigNotSupported:    true,
-			requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-linux-mbr-uefi-rhel-7",
-			// source created from projects/gce-uefi-images/global/images/ubuntu-1804-bionic-v20200317 and converted from GPT to MBR
-			source:                  "gs://compute-image-tools-test-resources/uefi/linux-ubuntu-mbr-uefi.vmdk",
-			os:                      "ubuntu-1804",
-			osConfigNotSupported:    true,
-			requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-windows-uefi-windows",
-			// source created from projects/gce-uefi-images/global/images/windows-server-2019-dc-core-v20200609
-			source:                  "gs://compute-image-tools-test-resources/uefi/windows-uefi-2019.vmdk",
-			os:                      "windows-2019",
-			requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		}, {
-			caseName: "inspect-uefi-windows-nonuefi-windows",
-			// source created from projects/windows-cloud/global/images/windows-server-2019-dc-v20200114
-			source:                    "gs://compute-image-tools-test-resources/uefi/windows-nonuefi-2019.vmdk",
-			os:                        "windows-2019",
-			notAllowedGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
-		},
-	*/
+	{
+		caseName: "inspect-uefi-linux-uefi-rhel-7",
+		// source created from projects/gce-uefi-images/global/images/rhel-7-v20200403
+		source:                  "gs://compute-image-tools-test-resources/uefi/linux-uefi-rhel-7.vmdk",
+		os:                      "rhel-7",
+		requiredGuestOsFeatures: []string{"UEFI_COMPATIBLE"},
+	}, {
+		caseName: "inspect-uefi-linux-uefi-rhel-7-from-image",
+		// image created from projects/gce-uefi-images/global/images/rhel-7-v20200403 and removed UEFI_COMPATIBLE
+		source:                  "projects/compute-image-tools-test/global/images/linux-uefi-no-guestosfeature-rhel7",
```

Comment:
In addition, can you add test cases to [1]? That allows you to assert the inspection results themselve:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/ef8b0df743b71aa0c1edee7973277b1bbaea08e0/cli_tools_e2e_test/diskinspect/disk_inspect_test.go#L57